src/pages/components/data-table/usage.mdx

```diff
@@ -12,8 +12,11 @@ Data tables are used to organize and display data efficiently. The data table co
 
 <AnchorLinks>
 
-<AnchorLink>Basic data table</AnchorLink>
-<AnchorLink>Additional functionality</AnchorLink>
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Behavior</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Related</AnchorLink>
```

Comment:
It would be good if there was a "References" section of validated articles under Related.